opendcs-rest-api/src/main/java/org/opendcs/odcsapi/res/AppResources.java

```diff
@@ -308,9 +309,9 @@ public void procFinished(String procName, Object obj, int exitStatus)
 		}
 		catch (IOException ex)
 		{
-			ex.printStackTrace();
-			throw new WebAppException(ErrorCodes.DATABASE_ERROR,
-				"Error attempting to start appId=" + appId + ": " + ex);
+			Logger.getLogger(ApiConstants.loggerName).log(Level.WARNING,
```

Comment:
Add the exception to the WebAppException overload and remove the redundant log message